serverlessworkflow/modules/ROOT/pages/persistence/postgresql-flyway-migration.adoc

```diff
@@ -0,0 +1,51 @@
+= Migrate Your Postgresql database with Kogito upgrade
```

Comment:
If this is a procedure module then add gerund ( the ing form t the end of the verb). Just a suggestion. 
```suggestion
= Migrating Your Postgresql database with the Kogito upgrade
```